### PR TITLE
Created README.md file with code snippets for the main library functionalities.

### PR DESCRIPTION
**Pull request by ognian-baruh**
Signed-off-by: Ognian Baruh <fixed-term.Ognyan.Baruh@bosch.io>

### CODE REVIEW

**Comment by ognian-baruh** [comment]
@thjaeckle please don't merge this PR until a member of the team has approved it.

File: README.md [hunk 1]
```diff
@@ -5,3 +5,312 @@
...
+        time.sleep(5)
+    except KeyboardInterrupt:
+        print("finished")
+        client.disconnect()
+        sys.exit()
+```
+
+Full example of the basic client connection can be found [here](examples/client_connect.py).
+
+### Creating a client instance as a class
```
Comment (konstantina-gramatova) [inline README.md:62]: To keep the README crisp - this section (class implementation approach) could be just mentioned as a usage possibility with a reference to the full example but not as a whole section.

The key info in this example is actually the possibility to use an external Paho client.

File: README.md [hunk 2]
```diff
@@ -5,3 +5,312 @@
...
+With these configurations a client instance can be created.
+
+```python
+client = Client(on_connect=on_connect, on_disconnect=on_disconnect)
+```
+
+After the client is created, it's ready to be connected.
+
+```python
+client.connect("localhost")
```
Comment (konstantina-gramatova) [inline README.md:50]: In difference to the Go library, the connection configurations here are provided per connection not per client instance, thus, would be nice to have an example for using them - e.g. connecting to a specific port, using a specific keep_alive, etc. In this sense, the whole section below (using as a class) can be narrowed down to a config option mention.

File: README.md [hunk 3]
```diff
@@ -5,3 +5,312 @@
...
+
+    def on_disconnect(self, ditto_client: Client):
+        print("Ditto client disconnected")
+        self.unsubscribe(self.on_message)
+
+    def on_message(self, request_id: str, message: Envelope):
+        print("request_id: {}, envelope: {}".format(request_id, message.to_ditto_dict()))
+
+    def on_log(self, ditto_client: Client, level, string):
+        print("[{}] {}".format(level, string))
```
Comment (konstantina-gramatova) [inline README.md:114]: Could be narrowed down and included in the connection configuration options in the beginning.

File: README.md [hunk 4]
```diff
@@ -5,3 +5,312 @@
...
+
+def send_inbox_message():
+    # create message and envelope
+    live_message = Message(thing_id).inbox(message_subject).with_payload("some_payload")
+    live_message_envelope = live_message.envelope(response_required=True, correlation_id="example-correlation-id")
+    live_message_dict = live_message_envelope.to_ditto_dict()
+    live_message_json = json.dumps(live_message_dict)
+    
+    # publish message
+    paho_client.publish(topic=req_topic + message_subject, payload=live_message_json)
```
Comment (konstantina-gramatova) [inline README.md:295]: This is an example helper - please, leave it out.

**Review by konstantina-gramatova — CHANGES_REQUESTED** [review]

**Pull request merged**